sotodlib/preprocess/processes.py

```diff
@@ -860,16 +860,22 @@ class GlitchFill(_Preprocess):
 
     def __init__(self, step_cfgs):
         self.signal = step_cfgs.get('signal', 'signal')
-        self.flag_aman = step_cfgs.get('flag_aman', 'glitches')
-        self.flag = step_cfgs.get('flag', 'glitch_flags')
+        self.flag_aman = step_cfgs.get('flag_aman')
+        self.flag = step_cfgs.get('flag')
```

Comment:
Why did you remove the default options here?

Reply:
These arguments are unified now.